certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -388,8 +388,8 @@
         "tags": [
           "DocuSign"
         ],
-        "summary": "Create envelope using template (V2)",
-        "description": "Create a new envelope using a specified template.",
+        "summary": "Create Envelope using Template - Envelope Custom Fields",
+        "description": "Create a new envelope using a specified template with an option to specify envelope custom fields.",
```

Comment:
What I have noticed in my testing is, if an Account has envelope custom fields set up, it applies to all the templates and envelopes that account is creating. I am not sure if there is a way not to use envelope custom fields in certain scenarios for that Account. Have you tried this @sriharip-docusign ? Is it possible to send an envelope without custom fields even though an Account has created custom fields in the settings?

Reply:
I think it depends on whether the fields are required or optional, but I'll let Sri reply

Reply:
@angelga is right. 

By default , once the envelope is created, it is stamped with all custom fields (required , optional, hidden or not) and set to default empty value.

You can send an envelope as long as it is has, no, empty required custom fields.